### PR TITLE
feat: Logpush for worker script

### PR DESCRIPTION
**Pull request by bjorand**

### CODE REVIEW

**Comment by bjorand** [comment]
@jacobbednarz do we have a clean way of reading the logpush setting for a worker? I can't find anything in the API or in the SDK.
I see the logpush status in `/accounts/<account_id>/workers/services/<script_name>` but I don't know if it is the right way to retrieve it.

File: docs/resources/worker_script.md [hunk 1]
```diff
@@ -83,6 +83,8 @@ resource "cloudflare_worker_script" "my_script" {
 - `secret_text_binding` (Block Set) (see [below for nested schema](#nestedblock--secret_text_binding))
 - `service_binding` (Block Set) (see [below for nested schema](#nestedblock--service_binding))
 - `webassembly_binding` (Block Set) (see [below for nested schema](#nestedblock--webassembly_binding))
+- `logpush` (Boolean) Whether to enable [Logpush](https://developers.cloudflare.com/workers/platform/logpush/)
```
Comment (jacobbednarz) [inline docs/resources/worker_script.md:86]: this is a generated artifact. check out https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/contributing/registry-documentation.md for how to update these files.

**Comment by jacobbednarz** [comment]
> @jacobbednarz do we have a clean way of reading the logpush setting for a worker? I can't find anything in the API or in the SDK.
I see the logpush status in /accounts/<account_id>/workers/services/<script_name> but I don't know if it is the right way to retrieve it.

not to my knowledge. if it doesn't exist in the SDK/API docs, we won't add support for it here as the stability isn't yet ready for public use.

**Comment by kamilkisiela** [comment]
Any chance to get this in?

**Pull request closed without merging**